src/rm.js

```diff
@@ -45,7 +45,12 @@ function rmdirSyncRecursive(dir, force, fromSymlink) {
 
   // if was directory was referenced through a symbolic link,
   // the contents should be removed, but not the directory itself
-  if (fromSymlink) return;
+  if (fromSymlink) {
```

Comment:
As I understand, the issue is that `link_to_a_dir` refers to something, but `link_to_a_dir/` does not (because only directories may be optionally referred to with a trailing slash). But in both cases, `link_to_a_dir` references a directory, and so I would expect `fromSymlink` to be in `true` in both cases. 

It sounds like we should rename the variable, or we should add a comment explaining what it actually represents.